### DISCUSSION TITLE
specify the difference/similarity between stratis (given no options for redundancy currently) and raid 0

### DISCUSSION THREAD

**Discussion by ilippert** [stratis-storage/project#381] [Q&A] [answered]
If I understand it correctly, currently stratis does not allow to specify redundancy.
If I combine several block devices in a pool, are reads/writes then distributed across both devices? making that explicit would help some users make decisions about how to design pools.
a practical question might be: what is the effect of combining in one pool, several block devices of different sizes, beyond adding available space for the pool?

addressing such questions relative to raid designs, would be helpful.

**Comment by drckeefe** [comment, marked as answer]
Thank you for your question, ilippert.
Adding additional devices to a pool provides more available space.  Devices are concatenated together to extend the pool's size. In a new pool, data is written to the filesystem(s), free blocks are consumed form the thin-pool in sequential order until all of the free blocks are used for that device, and then writes will proceed to the next device in the pool set.    RAID 0 on the other hand will stripe IO across known devices in the set.  Consider creating a RAID 0 using either hardware or software RAID and then creating a Stratis pool from the resulting device, if splitting IO is required in the workload.

**Comment by ilippert** [comment]
Thank you very much for the explanation, drckeefe.

This suffice absolutely. I would suggest to make this explicit in the stratis doc, readily recognisable. If such a documentation is not going to happen, I would be happy for the ticket to be closed.

**Comment by mulkieran** [comment]
We do not yet have a Stratis user manual, which is where this would belong, properly speaking. Moving to our Discussions section.
